guide/sequelize/currency.md

```diff
@@ -342,7 +342,7 @@ await user.addItem(item);
 message.channel.send(`You've bought: ${item.name}.`);
 ```
 
-In order for users to search for an item without having to care about case, we use the `$iLike` modifier when we're looking for the name. Keep in mind that this may be slow if you have millions of items, so please don't put a million items in your shop.
+For users to search for an item without caring about the case, we use the `$iLike` modifier when looking for the name. Keep in mind that this may be slow if you have millions of items, so please don't put a million items in your shop.
```

Comment:
```suggestion
For users to be able to search for an item without caring about the letter casing, we use the `$iLike` modifier when looking for the name. Keep in mind that this may be slow if you have millions of items, so please don't put a million items in your shop.
```